src/Brahma.FSharp.OpenCL.Workflow/Workflow.fs

```diff
@@ -0,0 +1,17 @@
+module Brahma.FSharp.OpenCL.Workflow.Workflow
+
+open Brahma.OpenCL
+open Brahma.FSharp.OpenCL.Workflow.BasicMonads.Reader
+
+type OpenCLContext(provider: ComputeProvider) =
+    member this.Provider = provider
+
+    member this.CommandQueue =
+        new CommandQueue(provider, provider.Devices |> Seq.head)
```

Comment:
Is it always safe to take head here?

Reply:
I have checked it and now in case of empty devices exception is thrown.